source/_docs/redirects.md

```diff
@@ -100,13 +103,14 @@ $redirects = array(
 if ((in_array($_SERVER['REQUEST_URI'], $redirects)) && (php_sapi_name() != "cli")) {
   header('HTTP/1.0 301 Moved Permanently');
   header('Location: https://'. $_SERVER['HTTP_HOST'] . '/new-path-for-all');
-  exit();
 
   # Name transaction "redirect" in New Relic for improved reporting (optional)
   if (extension_loaded('newrelic')) {
     newrelic_name_transaction("redirect");
   }
-}
```

Comment:
@dficker is this an intentional removal of the closing bracket, or a typo?

Reply:
Updated code in a new commit to fix the code issue.